src/app/pages/HomePage/ParaTimeSelector/para-time-selector-utils.ts

```diff
@@ -20,6 +20,19 @@ export abstract class ParaTimeSelectorUtils {
     }
   }
 
+  static showMobileHelpScreen(step: ParaTimeSelectorStep, isMobile: boolean) {
+    if (!isMobile) {
+      return false
+    }
+
+    switch (step) {
```

Comment:
why are we using switch statement to handle boolean ? Can't we just write 
`step === ParaTimeSelectorStep.ShowHelpScreen`. Additionally, do we even need utils for such simple case?

Reply:
I thought that mobile version would be more complex. Simplified the simple `switch` statements to `if`s, but I would keep the utils. If you feel strongly against I can move it into component.